src/app/patient-dashboard/common/todays-vitals/todays-vitals.component.spec.ts

```diff
@@ -30,34 +24,248 @@ import {
+  encounters: [
+    {
+      encounterDatetime: '2019-07-02T09:16:09.000+0300',
+      encounterType: {
+        display: 'HIV TRIAGE',
+        uuid: 'a44ad5e2-b3ec-42e7-8cfa-8ba3dbcf5ed7'
+      },
+      form: {
+        name: 'AMPATH POC Triage Encounter Form v1.2',
+        uuid: '0c2088e2-bf82-4ea8-b5f4-1a02de6976ca'
+      },
+      location: {
+        display: 'MTRH Other',
+        uuid: '2a901f42-bf51-428c-8be3-a135075abcb2'
+      },
+      patient: {
+        uuid: 'ce1b2ab4-6cb6-4483-a96d-d4b26ef54f0e'
+      }
+    }
+  ]
+});
 
-  let vitalsService: TodaysVitalsService, patientService: PatientService,
-    fakeAppFeatureAnalytics: AppFeatureAnalytics, component;
+const testVitals = [
+  {
+    color: '',
+    compoundValue: {
+      color: '',
+      compoundValue: null,
+      isCompoundedWith: 'systolic',
+      label: 'Diastolic',
+      name: 'diastolic',
+      order: 100,
+      show: true,
+      value: 60
+    },
+    isCompoundedWith: null,
+    label: 'BP:',
+    name: 'systolic',
+    order: 1,
+    show: true,
+    value: 120
+  },
+  {
+    color: '',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'Pulse:',
+    name: 'pulse',
+    order: 2,
+    show: true,
+    value: 41
+  },
+  {
+    color: 'red',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'Oxygen Saturation:',
+    name: 'oxygenSaturation',
+    order: 3,
+    show: true,
+    value: 5
+  },
+  {
+    color: 'red',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'Temperature:',
+    name: 'temperature',
+    order: 3,
+    show: true,
+    value: 35
+  },
+  {
+    color: '',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'Height:',
+    name: 'height',
+    order: 4,
+    show: true,
+    value: 173
+  },
+  {
+    color: undefined,
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'Weight:',
+    name: 'weight',
+    order: 5,
+    show: true,
+    value: 76
+  },
+  {
+    color: '',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'BMI(Kg/M2):',
+    name: 'bmi',
+    order: 6,
+    show: false,
+    value: '25.4'
+  },
+  {
+    color: '',
+    compoundValue: null,
+    isCompoundedWith: 'systolic',
+    label: 'Diastolic',
+    name: 'diastolic',
+    order: 100,
+    show: true,
+    value: 60
+  },
+  {
+    color: 'red',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'Weight for Height:',
+    name: 'weightForHeight',
+    order: 100,
+    show: false,
+    value: -4
+  },
+  {
+    color: '',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'Height for Age:',
+    name: 'heightForAge',
+    order: 100,
+    show: false,
+    value: '0'
+  },
+  {
+    color: 'red',
+    compoundValue: null,
+    isCompoundedWith: null,
+    label: 'BMI For Age:',
+    name: 'bmiForAge',
+    order: 100,
+    show: true,
+    value: '1'
+  }
+];
 
-  let fixture, el, patientServiceSpy;
+class PatientServiceStub {
```

Comment:
can we place the stub in a different file? For instance patient-service-stub.ts

Reply:
This being such a small object I don't think it merits moving into its own file

Reply:
Wouldn't we have other services in the future which might require us to stub?

Reply:
Possibly. But in this scenario I don't feel that these three stubs are anywhere near large enough to merit breaking them out into their own individual files.